initial commit

dustinsnoap opened the pull request:
pulled

dustinsnoap committed:
initial commit

dustinsnoap committed:
completed html and css for footer

dustinsnoap committed:
Header and Navigation HTML/CSS
Changed CSS headers to better stand out and seperate code

dustinsnoap committed:
Completed main content in index.html
Created first mixin: .button()

dustinsnoap committed:
Updated LESS with mixins and variables simplicity and readability

dustinsnoap committed:
Completed Mobile for index.html

dustinsnoap committed:
Reworked Header html/less for desktop/mobile
Completed Header on services.html

dustinsnoap committed:
Refactored LESS for first half of main

dustinsnoap committed:
Updated LESS for mobile on main
Cleaned up LESS on main

dustinsnoap committed:
Refactored LESS for main on index.html
Cleaned up LESS on main
TODO: footer

dustinsnoap committed:
refactored footer LESS for desktop

dustinsnoap committed:
Finished LESS for footer mobile
Cleaned up LESS for footer

dustinsnoap committed:
Changed 'div.item' to 'article'
Gave article it's own less
Created a tab.less for all tabs
adjusted articles to better fit services page
finished services.html for both desktop and mobile

dustinsnoap committed:
Built custom tab component for services.html
Removed all articles from services.html
Tab component creates, adds, and removes articles from dom

dustinsnoap committed:
MVP
Built and styled navigation system

dustinsnoap committed:
Added contact.html html and less for desktop

gsamaniego41 commented:
## MVP Features:
These MVP features represent the fundamental knowledge you should have obtained throughout the last 4 weeks.  If you cannot accomplish all of the MVP tasks listed here, you may be a candidate for the mastery based program.
* [x] Build layout **HTML/LESS**: [Home page desktop design](/DesignFiles/Home/home-desktop.png)
* [x] Build layout **HTML/LESS**: [Home page mobile design](/DesignFiles/Home/home-mobile.png)
* [x] Build layout **HTML/LESS**: [Services page **desktop design only**](/DesignFiles/Services/services-desktop.png)
* [x] Build layout **HTML/LESS/JavaScript**: [Navigation system design](DesignFiles/Navigation)
	* [Non-expanded View Design](DesignFiles/Navigation/non-expanded/non-expanded.png): When the navigation is not active, it should be fixed to the top of the page at all times
	* [Expanded View Design](DesignFiles/Navigation/expanded): When a user clicks on the navigation icon a full page overlay will pop over all the content on the page. The navigation icon will be replaced by a close icon. The user must click the close icon to return back to browsing the website. The home and services pages need to be linked together using `<a></a>` tags. You must use your own custom JavaScript for the opening and closing functionality, no bootstrap allowed for functionality on this feature
* [x] Build Custom Component **HTML/LESS/JavaScript**: [Services page tab navigator design](/DesignFiles/Services/tabs)
	* [Using the assets provided](/img/services), you must build the tab navigator using your own custom JavaScript component, no libraries allowed on this feature. You are encouraged to use code from previous weeks to accomplish this feature

dustinsnoap committed:
Fixed nav in index.html

dustinsnoap committed:
Fixed z-index for h2 of articles

dustinsnoap committed:
Completed LESS for contact.html mobile

dustinsnoap committed:
Completed html and less for carousel stretch

dustinsnoap committed:
Completed carousel js

dustinsnoap committed:
Added about.html and styles with less for both desktop and mobile
Started refining process for article.less and main.less
--temporariliy broke every page in the process

dustinsnoap committed:
Finished html and less for projects.html | desktop and mobile

dustinsnoap committed:
Organized the files a bit
Started fixing everything

dustinsnoap committed:
Finished JS for projects.html

dustinsnoap committed:
Fixed aboutus.html

dustinsnoap committed:
Fixed index.html
Updated to display: grid
No horizontal scroll bars!

dustinsnoap committed:
Fixed services.html
Updated to display: grid
Updated jumbo to carousel

dustinsnoap committed:
Fixed contact.html
Updated everything to grid

dustinsnoap committed:
Updated footer with grid

dustinsnoap committed:
Updated carousel less and js
Fixed issues where arrows weren't updating class on selected tab

dustinsnoap committed:
Minor fixes and updates
Updated the navigation menu to include all pages
Updated links in navigation to direct to correct pages
Updated logo to direct back to home
